project/make/.dockerinit_gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+IAMSTATIC="true"
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+EXTLDFLAGS_STATIC_DOCKER="-static -Wl,--no-export-dynamic $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
```

Comment:
I agree -- also, why the hairy `echo | sed` logic?  Isn't the only change from that to remove `-static` (which is then added back here)?